cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -497,6 +508,20 @@ open class Node @Inject constructor(private val project: Project) {
         }
     }
 
+    internal fun installPostgresDriver() {
+
+        val driverJar = project.configuration("cordapp").files {
+            (it.group == "org.postgresql") &&
+                    (it.name == "postgresql") &&
+                    (it.version == "42.2.10")
```

Comment:
This is the sort of case where having the version number as a constant would be valuable.